/opt/conda/lib/python3.9/site-packages/datajoint/connection.py:341: UserWarning: MySQL server has gone away. Reconnecting to the server.
  warnings.warn("MySQL server has gone away. Reconnecting to the server.")
Connecting datajoint-worker@rds.datajoint.io:3306
EField already populated
DField already populated
['podman', '--version', '']
using podman version: 3.4.2
podman logs docker_standard_worker_1
exit code: 0
